roborock/api.py

```diff
@@ -474,6 +476,12 @@ async def code_login(self, code) -> UserData:
         if response_code != 200:
             if response_code == 2018:
                 raise RoborockInvalidCode("Invalid code - check your code and try again.")
+            if response_code == 3009:
+                raise RoborockNoUserAgreement("You must accept the user agreement in the Roborock app to continue.")
+            if response_code == 3006:
+                raise RoborockInvalidUserAgreement(
+                    "User agreement must be accepted again - or you are attempting to use the Mi Home app."
```

Comment:
"User agreement must be accepted again - or you are attempting to use the Mi Home app account." maybe?